frontend/src/app/data/User.js

```diff
@@ -19,6 +21,7 @@ export async function fetchUserData(userId) {
  * Retrieves data directly from Firebase, requires user to be authenticated.
  */
 export function getUserData(userId) {
+    getAnalytics()
```

Comment:
Don't import and call it in this file because this will only run if this function is called. Instead, you can import and call it in App.jsx.